contracts/utils/Strings.sol

```diff
@@ -438,4 +447,42 @@ library Strings {
             value := mload(add(buffer, add(0x20, offset)))
         }
     }
+    
+    function _escapeJsonString(string memory input) private pure returns (string memory) {
+    bytes memory buffer = bytes(input);
+    bytes memory output = new bytes(buffer.length * 2); // Allocate max possible space
+    uint256 outputLength = 0;
+
+    for (uint256 i; i < buffer.length; ) {
+        bytes1 char = buffer[i];
+
+        if ((SPECIAL_CHARS_LOOKUP & (1 << uint8(char))) != 0) {
+            output[outputLength++] = '\\';
+
+            if (char == 0x08) output[outputLength++] = 'b';
+            else if (char == 0x09) output[outputLength++] = 't';
+            else if (char == 0x0A) output[outputLength++] = 'n';
+            else if (char == 0x0C) output[outputLength++] = 'f';
+            else if (char == 0x0D) output[outputLength++] = 'r';
+            else if (char == 0x22) output[outputLength++] = '"';
+            else if (char == 0x2F) output[outputLength++] = '/';
+            else if (char == 0x5C) output[outputLength++] = '\\';
+        } else {
+            output[outputLength++] = char;
+        }
+
+        unchecked { ++i; }
```

Comment:
Please lets all stop doing that. The compiler knows how to handle this correctly.

Reply:
On the other hand, its worth putting the entier function body in an unchecked block so that all the `outputLength++` are unchecked.